### ISSUE TITLE
[0.3.0] Deployment to Netlify broken

### ISSUE THREAD

**Issue by olance** [redwoodjs/redwood#337]
After migrating the tutorial project to Redwood 0.3.0, the deployment to Netlify is broken.
Queries to the graphql function fail with the error:

```
0: "Runtime.ImportModuleError: Error: Cannot find module '/opt/build/repo/api/src/lib/db'"
1: "Require stack:"
2: "- /var/task/src/api/dist/services/contacts/contacts.js"
3: "- /var/task/src/api/dist/functions/graphql.js"
4: "- /var/task/graphql.js"
5: "- /var/runtime/UserFunction.js"
6: "- /var/runtime/index.js"
7: "    at _loadUserApp (/var/runtime/UserFunction.js:100:13)"
8: "    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)"
9: "    at Object.<anonymous> (/var/runtime/index.js:43:30)"
10: "    at Module._compile (internal/modules/cjs/loader.js:1158:30)"
11: "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)"
12: "    at Module.load (internal/modules/cjs/loader.js:1002:32)"
13: "    at Function.Module._load (internal/modules/cjs/loader.js:901:14)"
14: "    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)"
15: "    at internal/main/run_main_module.js:18:47"
```

In the built files generated by `yarn rw build`, I can see that services are required with relative paths from `api/dist/functions/graphql.js`.
However in the services themselves, the `require` call to `db` uses an absolute path (here on a local build of course):

```
var _db = require("/Users/olance/Documents/Dev/redwoodblog/api/src/lib/db");
```

Looks like that could be the issue? ^^

**Comment by peterp** [comment]
@olance could you share your repo with us?

**Comment by peterp** [comment]
@olance I've got a local reproduction going, we'll fix it shortly

**Comment by thedavidprice** [comment]
@olance We'll merge and publish the fix into v0.3.2

But possibly you could try the fix locally in parallel and confirm it deploys ok?
https://github.com/redwoodjs/redwood/pull/338/files

**Comment by olance** [comment]
@peterp sorry, had forgotten the link to my repo!

Just deployed to Netlify and things are now working perfectly!

Thanks :)
